api/Dockerfile

```diff
@@ -19,11 +19,11 @@ ENV PIP_NO_COLOR=1
 # - Create a virtualenv inside `/venv`
 # - Install PDM to install Python dependencies
 RUN apt-get update \
-  && apt-get install -yqq --no-install-recommends \
-    python3-dev \
+  && apt-get install -yqq --no-install-recommends python3-dev \
   && apt-get autoremove -y \
   && rm -rf /var/lib/apt/lists/* \
-  && pip install pdm~=2.19
+  && pip install --upgrade pip \
+  && pip install pdm~=2.21
```

Comment:
We have the Python version defined as `API_PY_VERSION` on Line 5, does it maybe make sense to make this package version an similar argument in the same place in the Dockerfile? Even if it just has a default to the pinned version.

Reply:
That makes sense, it would also make it easier to bump PDM everywhere by changing it in one place!